flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/gateway/context/SessionContextTest.java

```diff
@@ -232,24 +229,25 @@ private Configuration getConfiguration() {
 
     private void validateAddJar(String jarPath) throws IOException {
         sessionContext.addJar(jarPath);
-        assertEquals(Collections.singletonList(udfJar.getPath()), sessionContext.listJars());
-        assertEquals(
-                Collections.singletonList(udfJar.toURI().toURL().toString()),
-                getConfiguration().get(JARS));
+        assertThat(sessionContext.listJars())
+                .isEqualTo(Collections.singletonList(udfJar.getPath()));
```

Comment:
containsExactly?

Reply:
`containsExactly` is a vararg. A good conversion here could be:

```
        assertThat(sessionContext.listJars())
                .containsOnly(udfJar.getPath());
```